habitat_baselines/rl/hrl/skills/nav.py

```diff
@@ -77,7 +77,7 @@ def _is_skill_done(
         return should_stop
 
     def _parse_skill_arg(self, skill_arg):
-        targ_name, targ_idx = skill_arg[-1].split("|")
+        targ_name, targ_idx = skill_arg[-2].split("|")
```

Comment:
Was this a bug or did the code change in such a way that the argument position changed. 
In either case, can you add a unit test?

Reply:
Not a bug before, I changed the code so that this argument was changed. This is already tested in the composite task tests.